resources/constants.rb

```diff
@@ -5,6 +5,22 @@ def self.AssumedInsideTemp
     return 73.5 # deg-F
   end
 
+  def self.C_d_clg_low_seer
+    return 0.20
+  end
+
+  def self.C_d_clg_high_seer
+    return 0.07
+  end
+
+  def self.C_d(var_speed = false)
```

Comment:
Can you merge all three methods into a single C_d with the appropriate arguments?